### ISSUE TITLE
wifi cant get hdcp ip

### ISSUE THREAD

**Issue by WangChun420** [meatpiHQ/wican-fw#8]
when ble disconnect , wifi restart. but cant connect it ,seems not hdcp
W (26804) wifi_network: wifi restart
I (26808) config_server_restart: Starting server on port: '80'
I (26816) config_server_restart: Registering URI handlers
I (26823) BLE: advertising start success
I (27766) app_main: free heap : 13952
I (29766) app_main: free heap : 13620
I (31766) app_main: free heap : 13952
I (33766) app_main: free heap : 13952
I (33842) wifi:new:<6,0>, old:<6,0>, ap:<6,0>, sta:<255,255>, prof:6
I (33842) wifi:station: 82:0a:1b:cb:5d:65 join, AID=1, bgn, 20
W (34072) wifi:<ba-add>idx:2 (ifx:1, 82:0a:1b:cb:5d:65), tid:0, ssn:0, winSize:64
W (34984) wifi:<ba-add>idx:3 (ifx:1, 82:0a:1b:cb:5d:65), tid:6, ssn:2, winSize:64

**Comment by meatpiHQ** [comment]
Yes, I'm aware of this issue, I suspect this is related to the SDK somehow. In the latest firmware release, when the BLE connects the WiFi AP will be disabled until the next power cycle.
